index.html

```diff
@@ -4928,13 +4928,21 @@ <h3>Fitness for Purpose</h3>
     <section class="appendix informative">
       <h2>Contexts, Types, and Credential Schemas</h2>
 
-      <section class="informative">
+      <section class="normative">
         <h3>Base Context</h3>
+
+        <p class="issue" title="(AT RISK) Hash values might change during Candidate Recommendation">
+This section lists cryptographic hash values that might change during the
+Candidate Recommendation phase based on implementer feedback that requires
+the referenced files to be modified.
+        </p>
           <p>
-The base context, located at <code>https://www.w3.org/ns/credentials/v2</code>
-with a SHA-256 digest of
-<strong><code>ff1fdfa8d3f07547ec149b6652fe3f5a4276b9f0c48344dc29400763ba7b44fa</code></strong>,
-can be used to implement a local cached copy. It is possible to confirm the
+Implementations MUST ensure that the base context value, located at
+<code>https://www.w3.org/ns/credentials/v2</code>, matches the following
+SHA-256 digest of the value:
+<strong><code>dc28cf86d8a3199beb9d14e29d1da662346b8660abf2edca873098c88ca2a457</code></strong>.
+The base context value matching the digest previously stated can be used to
+implement a local cached copy. It is possible to confirm the
 SHA-256 digest by running the following command from a modern Unix command
 interface line:
 `curl -s https://www.w3.org/ns/credentials/v2 | openssl dgst -sha256 -hex -r`.
```

Comment:
```suggestion
The base context value matching the digest previously stated can be used to
implement a local cached copy. It is possible to confirm the
SHA-384 digest by running the following command from a modern Unix command
interface line:
`curl -s https://www.w3.org/ns/credentials/v2 | openssl dgst -sha384 -binary | openssl base64 -A`.

More details regarding this hash encoding and methods may be found in the <a href="https://www.w3.org/TR/SRI/#integrity-metadata">integrity metadata</a> sectoion of [[SRI]].
```

Reply:
@mprorock — `s/sectoion/section/` in line 4950 (which should also become 3 lines, to conform with other line lengths)